src/OrbitGl/App.cpp

```diff
@@ -1154,8 +1154,9 @@ void OrbitApp::LoadSymbols(const std::filesystem::path& symbols_path, ModuleData
                                          module_data->file_path(), symbols_path.string()));
   thread_pool_->Schedule([this, scoped_status = std::move(scoped_status), symbols_path, module_data,
                           function_hashes_to_hook = std::move(function_hashes_to_hook),
-                          frame_track_function_hashes =
-                              std::move(frame_track_function_hashes)]() mutable {
+                          frame_track_function_hashes = std::move(frame_track_function_hashes),
+                          main_thread_executor =
+                              main_thread_executor_->weak_from_this()]() mutable {
```

Comment:
looks like main_thread_executor is not used in lambda.

Reply:
Hmmm. You're right. This is from a different PR. Let me quickly fix that!

Reply:
Done. Thanks for spotting this!